BraintreeCore/src/main/java/com/braintreepayments/api/HttpNoTimingResponse.kt

```diff
@@ -0,0 +1,7 @@
+package com.braintreepayments.api
+
+internal class HttpNoTimingResponse : HttpTimingResponseCallback {
```

Comment:
Are we using this anywhere?

Reply:
Yep, Even Steven asked if we could make the callback optional, but it's something I need to investigate.
https://github.com/braintree/braintree_android/blob/d6ec39e79bf546e5ce57bd9cb460947e4acfe7e1/BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.kt#L192

Reply:
Removed [fa734e9](https://github.com/braintree/braintree_android/pull/1032/commits/fa734e964296b2dda10156edc4ccc0d271284fc1)